js/modules/k6/data/share.go

```diff
@@ -20,7 +20,7 @@ type wrappedSharedArray struct {
 	parse    sobek.Callable
 }
 
-func (s sharedArray) wrap(rt *sobek.Runtime) sobek.Value {
+func (s sharedArray) Wrap(rt *sobek.Runtime) sobek.Value {
```

Comment:
Nit; after a quick look, I haven't been able to identify the reason why need to expose this. If there's a reason, I guess it's but, but if there isn't, I'd prefer to keep it private so we reduce the surface. Especially considering the type is also a private one.

Reply:
Good catch 👀 I'll give a shot at making it private then 👍🏻